print/locales/fr-CH-scout.json

```diff
@@ -1 +1,7 @@
-{}
+{
```

Comment:
And this complete files should probably be empty. I think there was a mistake between source and destination.